src/main/java/net/greatstart/controllers/UserController.java

```diff
@@ -1,14 +1,19 @@
 package net.greatstart.controllers;
 
 
+import net.greatstart.dto.DtoUser;
 import net.greatstart.dto.DtoUserProfile;
 import net.greatstart.mappers.UserProfileMapper;
 import net.greatstart.model.User;
+import net.greatstart.services.SecurityService;
 import net.greatstart.services.UserService;
+import net.greatstart.validators.UserValidationService;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.greatstart.validators.UserValidationService'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)